backend/db/utils.py

```diff
@@ -565,9 +565,20 @@ def run_sql(con: Connection, command: str, params: Dict = {}) -> Any:
     return q
 
 
-def show_tables(con: Connection = None, print_dump=True) -> List[Row]:
-    """Show tables"""
-    conn = con if con else get_db_connection(schema=SCHEMA)
+# todo: should add a 'include_views' param because it is ambiguous as to whether or not this returns views. it does.
+def list_schema_objects(
```

Comment:
## `show_tables` Updates
- Renamed to `show_schema_objects`
- Added `schema` param
- Merged `list_tables` unique functionality into here, as below
- `list_tables` now just calls this and passes fixed params
- `list_views` now just calls this and passes fixed params
- Added filters for: views, sequences, tables, temp refresh tables/views (i.e. `*_old`, `*_new`)
- Added `names_only` param to return simple list of strings